pkgs/by-name/li/livekit-cli/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
```

Comment:
One commit per package, in topological (dependencies before dependents) order.

Reply:
I've updated the branch to split the two packages in two different commits. They're not dependent on each other, so I've put `livekit` before `livekit-cli`, as you need a server to manage with the CLI